CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ PROJECT(lightgbm)
 
 OPTION(USE_MPI "MPI based parallel learning" OFF)
 OPTION(USE_OPENMP "Enable OpenMP" ON)
-OPTION(USE_GPU "Enable GPU-acclerated training (EXPERIMENTAL)" OFF)
+OPTION(USE_GPU "Enable GPU-accelerated training (EXPERIMENTAL)" OFF)
```

Comment:
BTW, @huanzhang12, is GPU version still **EXPERIMENTAL**?

Reply:
I think it has become stable enough. You can remove the word "EXPERIMENTAL"